sdk/communication/Azure.Communication.CallAutomation/tests/Infrastructure/CallAutomationClientAutomatedLiveTestsBase.cs

```diff
@@ -49,6 +49,7 @@ public CallAutomationClientAutomatedLiveTestsBase(bool isAsync, RecordedTestMode
             JsonPathSanitizers.Add("$..id");
             JsonPathSanitizers.Add("$..rawId");
             JsonPathSanitizers.Add("$..value");
+            JsonPathSanitizers.Add("$..botAppId");
```

Comment:
Should we add operationContext and ivrContext?

Reply:
I think it's okay but will get additional feedback